sunpy/data/_sample.py

```diff
@@ -7,6 +7,7 @@
 from sunpy.util.parfive_helpers import Downloader
 
 _BASE_URLS = (
+    'https://github.com/sunpy/data/raw/main/sunpy/v1/',
```

Comment:
Does sample data work without making these changes? If so I think we should either leave the sample data alone, or if making these changes put them in a single commit with a meaningful commit message.

Reply:
Without these sample data changes it should fall back to the data.sunpy.org endpoint when we rename the sample data repo.

Reply:
Given we are doing this, I would rather add this URL just to save me a few $ on the AWS bill. +1 on the history tho.